metricbeat/module/dropwizard/collector/_meta/data.json

```diff
@@ -0,0 +1,29 @@
+{
+    "@timestamp": "2017-04-16T06:42:37.441Z",
+    "beat": {
+        "hostname": "beathost",
+        "name": "beathost",
+        "version": "6.0.0-alpha1"
+    },
+    "dropwizard": {
+        "test": {
```

Comment:
As it is a "dynamic" metricset I'm a bit undecided what we should do here. In other dynamic metricsets we leave it empty here (https://github.com/elastic/beats/blob/master/metricbeat/module/windows/perfmon/_meta/data.json) but provide an example output in the docs.